spec/controllers/admin_outgoing_message_controller_spec.rb

```diff
@@ -55,6 +55,36 @@
 
     end
 
+    context 'unsuccessfully destroying the message' do
```

Comment:
...or leave the description and pull `allow_any_instance_of(OutgoingMessage).to receive(:destroy).and_return(false)` into a `before do` for the context block so it's easier to see what's about to happen